strawberry/utils/__init__.py

```diff
@@ -1,5 +1,4 @@
-import graphql
-from packaging.version import Version
+from graphql.version import VersionInfo, version_info
 
-IS_GQL_33 = Version(graphql.__version__) >= Version("3.3.0a")
+IS_GQL_33 = version_info >= VersionInfo.from_str("3.3.0a0")
```

Comment:
Ah nice! I think I like this more 😊

assuming graphql core will keep exporting VersionInfo :D

Reply:
Sure, always a concern. But given the tightness of the pin, it seems unlikely this project would be caught unaware.

Reply:
yup, I'd say let's merge this :)

can you add a RELEASE.md file?

```
release type: patch

This release updates how we check for GraphQL core's version to remove a dependency on the `packaging` package
```

something like this 😊